docs: rebranding changes (#3014)
chanzuckerberg/single-cell-data-portal: frontend/doc-site/04__Analyze Public Data/4_1__Hosted Tutorials.mdx

# CZ CellxGene Explorer Tutorials

> The CZ CellxGene Explorer enables exploration and interrogation of individual single cell data sets. This section describes CellxGene's data exploration capabilities.

Reviewer: ```suggestion
The CELLxGENE Explorer is a visual tool linked to each dataset hosted by CELLxGENE Discover. Explorer enables analysis and interrogation of individual single cell data sets. This section describes Explorer's capabilities.
```

## CZ CellxGene Explorer Interface

CellxGene Explorer's user interface organizes single cell data similarly to how it is organized in single cell data formats. The left hand side displays categorical and numerical sample metadata. The right hand side is a space for displaying features such as genes and gene sets. The center displays the embedding, where each cell is a point. UMAP and tSNE are common embeddings, which place cells based on their local distances in gene expression space. Cells from spatial data can also be displayed using each cell's (x, y) coordinates.

**Key Concepts**: User Interface Explanation

<EmbeddedGoogleSlides src="https://docs.google.com/presentation/d/e/2PACX-1vQzA99mYecRu0Y5v--QsUCAuoTJFxo7VIhApCB-E669-00KCKww8AumBZcX0pwV41qHIuLRTsL7AckB/embed?start=false&loop=false&delayms=3000" />

## Examining Categorical Metadata

> Categorical metadata (such as tissue of origin or cell type) can be used in a number of ways within CellxGene including coloring embedding plots (i.e. color UMAP by cell type), looking at cell counts, within a categorical metadata field, making selections of cells or viewing the interaction between different categorical metadata fields.

Reviewer: ```suggestion
Categorical metadata (such as tissue of origin or cell type) can be used in a number of ways within Explorer including coloring embedding plots (i.e. color UMAP by cell type), looking at cell counts, within a categorical metadata field, making selections of cells or viewing the interaction between different categorical metadata fields.
```

## Selecting and Subsetting Cells

> CellxGene allows for the complex selection of cells via selection directly on the embedding, gene expression cutoffs, and based on categorical metadata attributes.

Reviewer: ```suggestion
Explorer allows for the complex selection of cells via selection directly on the embedding, gene expression cutoffs, and based on categorical metadata attributes.
```

**Key Concepts**: Categorical Metadata Selection, Numerical Metadata Selection, Complex Selection (combining selection methods)

<EmbeddedGoogleSlides src="https://docs.google.com/presentation/d/e/2PACX-1vSDkm8hljo9zXIpI0kBCKRpOyKUJx8MN6lsOXw-OlxFy6b04g6DiY4HKREisl4kttfXHqP2kr7AG6pv/embed?start=false&loop=false&delayms=3000" />

## Compare Expression of Multiple Genes

> CellxGene allows you to compare the expression of multiple genes via bivariate plots.

Reviewer: ```suggestion
Explorer allows you to compare the expression of multiple genes via bivariate plots.
```

**Key Concepts**: Gene Expression, Co-expression, Cell Selection, Subsetting

<EmbeddedGoogleSlides src="https://docs.google.com/presentation/d/e/2PACX-1vQJ5623UX9tJJKNKDlqD1kBpL3mtbJ89MDpV7GAJHJd2DLr6qJJux4g9ZaeLcuTPvwUI1Q3jULDkYWh/embed?start=false&loop=false&delayms=3000" />

## Using Gene Sets to Learn About Cell Population Functional Characteristics

> CellxGene allows you to examine groups of genes via the gene sets feature.

Reviewer: ```suggestion
Explorer allows you to examine groups of genes via the gene sets feature.
```

## Find Marker Genes

> CellxGene allows you to find marker genes between selected cell populations.

Reviewer: ```suggestion
Explorer allows you to find marker genes between selected cell populations.
```
